job/job.go

```diff
@@ -231,6 +238,7 @@ func (j *Job) CallbackInfo(downloadURL url.URL) (Callback, error) {
 
 func (j Job) String() string {
 	return fmt.Sprintf("Job{ID:%s, Aggr:%s, URL:%s, callback_url:%s, "+
-		"callback_type:%s, callback_dst:%s, Timeout:%d, UserAgent:%s}",
-		j.ID, j.AggrID, j.URL, j.CallbackURL, j.CallbackType, j.CallbackDst, j.DownloadTimeout, j.UserAgent)
+		"callback_type:%s, callback_dst:%s, Timeout:%d, RequestHeaders:%#v}",
```

Comment:
Use the less verbose "%v" which does the job just fine.

Reply:
Used `%v`!